package.json

```diff
@@ -14,15 +14,14 @@
   "scripts": {
     "test": "jest",
     "test:ci": "jest --coverage",
-    "test:ci-coverage": "jest --coverage && cat ./coverage/lcov.info | codacy-coverage",
     "test:watch": "jest --watch",
     "test:update": "jest -u",
     "test:dev": "jest --watchAll -u --coverage",
-    "lint": "eslint --ext .js --ext .jsx src/ && eslint --ext .js --ext .jsx stories/",
```

Comment:
Ah, didn't realise this had been done, or why I didn't do this in the first place!!